### PR TITLE
FileWriter in tf.summary solved

### PR DESCRIPTION
**Pull request by ayushmankumar7**
Issue: #37113

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,13 +42,13 @@ To install the current release, which includes support for
 Windows)*:
 
 ```
-$ pip install tensorflow
+$ pip3 install tensorflow
```
Comment (mihaimaruseac) [inline README.md]: Let's not change this file. In several usecases `pip` is `pip3` already.

File: tensorflow/python/keras/losses.py [hunk 2]
```diff
@@ -646,7 +646,7 @@ class CategoricalHinge(LossFunctionWrapper):
   """Computes the categorical hinge loss between `y_true` and `y_pred`.
 
   `loss = maximum(neg - pos + 1, 0)`
-  where `neg = sum(y_true * y_pred)` and `pos = maximum(1 - y_true)`
+  where `neg = maximum((1 - y_true)*y_pred)` and `pos = sum(y_true * y_pred)`
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/losses.py]: This is unrelated change. Please don't include unrelated changes.

File: tensorflow/tools/api/tests/module_test.py [hunk 3]
```diff
@@ -70,15 +70,14 @@ def testCompatV2HasCompatV1(self):
 
   def testSummaryMerged(self):
     # pylint: disable=pointless-statement
-    tf.summary.image
+    tf.compat.v1.summary.image
```
Comment (mihaimaruseac) [inline tensorflow/tools/api/tests/module_test.py]: Unrelated change

File: tensorflow/tools/api/tests/module_test.py [hunk 4]
```diff
@@ -70,15 +70,14 @@ def testCompatV2HasCompatV1(self):
...
     # If we use v2 API, check for create_file_writer,
     # otherwise check for FileWriter.
     if hasattr(tf, '_major_api_version') and tf._major_api_version == 2:
-      tf.summary.create_file_writer
+      tf.compat.v1.summary.create_file_writer
     else:
-      tf.summary.FileWriter
+      tf.compat.v1.summary.FileWriter
```
Comment (mihaimaruseac) [inline tensorflow/tools/api/tests/module_test.py]: We should prefer V2 symbols.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ayushmankumar7) [reply README.md]: Sure. Then let me revert the changes

File: tensorflow/python/keras/losses.py (hunk 2 above)
Comment (ayushmankumar7) [reply tensorflow/python/keras/losses.py]: Okay

File: tensorflow/tools/api/tests/module_test.py (hunk 4 above)
Comment (ayushmankumar7) [reply tensorflow/tools/api/tests/module_test.py]: Sure. Changed

File: README.md [hunk 5]
```diff
@@ -61,7 +61,7 @@ commands.
 #### *Try your first TensorFlow program*
 
 ```shell
-$ python
+$ python3
```
Comment (mihaimaruseac) [inline README.md]: This one should also be reverted

File: tensorflow/python/keras/losses.py [hunk 6]
```diff
@@ -646,8 +646,7 @@ class CategoricalHinge(LossFunctionWrapper):
   """Computes the categorical hinge loss between `y_true` and `y_pred`.
 
   `loss = maximum(neg - pos + 1, 0)`
-  where `neg = sum(y_true * y_pred)` and `pos = maximum(1 - y_true)`
-
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/losses.py]: This change should not be in this PR

File: tensorflow/tools/api/tests/module_test.py [hunk 7]
```diff
@@ -74,11 +74,10 @@ def testSummaryMerged(self):
     # If we use v2 API, check for create_file_writer,
     # otherwise check for FileWriter.
     if hasattr(tf, '_major_api_version') and tf._major_api_version == 2:
-      tf.summary.create_file_writer
+      tf.compat.v2.summary.create_file_writer
```
Comment (mihaimaruseac) [inline tensorflow/tools/api/tests/module_test.py]: `tf.compat.v2` is TF 2.x but then `tf.` is the same API. Do we really need to change this line?

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: tensorflow/tools/api/tests/module_test.py (hunk 7 above)
Comment (ayushmankumar7) [reply tensorflow/tools/api/tests/module_test.py]: Required changes made.

File: README.md (hunk 5 above)
Comment (ayushmankumar7) [reply README.md]: Reverted.

File: tensorflow/python/keras/losses.py (hunk 6 above)
Comment (ayushmankumar7) [reply tensorflow/python/keras/losses.py]: I am sorry. I am unable to remove this commit as this was in the same branch.

**Comment by ayushmankumar7** [comment]
Lemme Solve this and create a  PR again.

File: tensorflow/python/keras/losses.py (hunk 6 above)
Comment (ayushmankumar7) [reply tensorflow/python/keras/losses.py]: I am Closing this PR and creating a fresh PR

**Comment by ayushmankumar7** [comment]
Closing this PR and creating a new one.

**Comment by ayushmankumar7** [comment]
Link to New PR : https://github.com/tensorflow/tensorflow/pull/37223
